### PR TITLE
Android talkback issue in Navigation

### LINKED ISSUE #2999: Scrollable Navigation - Android Talkback announces link twice when navigation scrolls
**Describe the bug**
If a navigation item begins off screen, and focusing that item causes it to be brought onto the screen, talkback announces the text within that item twice (once before the link is brought into view, and once after).

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'localhost:7080/afrique'
2. Using talkback, scroll through the scrollable navigation until you reach an item that is off screen.

**Expected behaviour**
The item should only be announced once.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by mmcculloch**
Resolves #2999 

**Overall change:** 
When using Android Talkback and moving between nav items `scroll-behavior: smooth;` was causing talkback to repeat the item 2-3 times because of the scroll speed. This change reverts it back to auto, with a comment to indicate it shouldn't be set to smooth.

A few other things that I checked and discovered
- This style is only applied in mobile view, so disabling it has no effect on Desktop.
- This style is not implemented in Safari, so there is no different in iOS Safari accessibility
- This style only has an effect when tabbing with accessibly settings, but generally it is considered back accessibility practice to enable it https://stackoverflow.com/questions/51152749/smooth-scrolling-while-navigating-with-tab-key

**Code changes:**

- CSS change to change `scroll-behavior: smooth;` to `scroll-behavior: auto;`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx [hunk 1]
```diff
@@ -15,7 +15,9 @@ const StyledScrollableNav = styled.div`
   @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
     white-space: nowrap;
     overflow-x: scroll;
-    scroll-behavior: smooth;
+
+    /* Avoid using smooth scrolling as it causes accessbility issues */
```
Comment (DenisHdz) [inline packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx]:
```suggestion
    /* Avoid using smooth scrolling as it causes accessibility issues */
```

**Review by DenisHdz — APPROVED** [review]
Thanks for investigating 👍

File: packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx (hunk 1 above)
Comment (mmcculloch) [reply packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx]: doh, I'm blaming my mac keyboard on this :)

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.6 | [PR#3177](https://github.com/bbc/psammead/pull/3252) Remove `scroll-behavior: smooth;` to from StyledScrollableNav |
```
Comment (j-pendlebury) [inline packages/components/psammead-navigation/CHANGELOG.md]:
```suggestion
| 6.0.6 | [PR#3252](https://github.com/bbc/psammead/pull/3252) Remove `scroll-behavior: smooth;` from StyledScrollableNav |
```

**Review by j-pendlebury — APPROVED** [review]
Just one thing, otherwise looks great 👍

**Pull request merged**